### ISSUE TITLE
Missiles sometimes going through target without hitting it.

### ISSUE THREAD

**Issue by evertvorster** [vegastrike/Vega-Strike-Engine-Source#344]
Seems that the collision detection code needs some love. 

Chances of a dumbfire hitting Oswald when fired at point blank range is less than even.... 
I have heard reports of the chances being only 10 percent, but I have been having better luck than that...

**Comment by stephengtuggy** [comment]
Thanks for filing these tickets.

**Comment by royfalk** [comment]
Yes. This is a known problem that I suspect got worse in one of my refactorings.
Can you do a test - change the speed of the missile to that of a ship (say 500) and see if that improves the odds.
I suspect what happens is that the game evaluates collision every ATOM and the missile is so fast it skips over the ship.
One way to fix this is to increase the radius of the missile and make it a proximity fuse.
Another would be to actually fix the collider but that's much harder.

**Comment by stephengtuggy** [comment]
> I suspect what happens is that the game evaluates collision every ATOM and the missile is so fast it skips over the ship.

That makes a lot of sense.
